packages/wrangler/src/config/validation-helpers.ts

```diff
@@ -533,6 +533,31 @@ export const validateAdditionalProperties = (
 	return true;
 };
 
+/**
+ * Check that any configurations that include both cron triggers and smart placement, don't
+ */
+export const validateSmartPlacementConfig = (
+	diagnostics: Diagnostics,
+	placement: { mode: "off" | "smart" } | undefined,
+	triggers:
+		| {
+				crons: string[];
+		  }
+		| undefined
+): boolean => {
+	if (
+		placement?.mode === "smart" &&
+		triggers?.crons &&
+		triggers.crons.length > 0
```

Comment:
NIT - I think you can just do

```suggestion
		triggers?.crons?.length > 0
```

Reply:
Updated to `placement?.mode === "smart" && !!triggers?.crons?.length`